src/PDE_BPINN.jl

```diff
@@ -295,7 +295,57 @@ function inference(samples, pinnrep, saveats, numensemble, ℓπ)
     end
 end
 
-# priors: pdf for W,b + pdf for ODE params
+"""
+```julia
+ahmc_bayesian_pinn_pde(pde_system, discretization;
+        draw_samples = 1000,
+        bcstd = [0.01], l2std = [0.05],
+        phystd = [0.05], priorsNNw = (0.0, 2.0),
+        param = [], nchains = 1, Kernel = HMC(0.1, 30),
+        Adaptorkwargs = (Adaptor = StanHMCAdaptor,
+            Metric = DiagEuclideanMetric, targetacceptancerate = 0.8),
+        Integratorkwargs = (Integrator = Leapfrog,), saveats = [1 / 10.0],
+        numensemble = floor(Int, draw_samples / 3), progress = false, verbose = false)               
+```
+## NOTES 
+* Dataset is required for accurate Parameter estimation + solving equations.
+* Returned solution is a BPINNsolution consisting of Ensemble solution, estimated PDE and NN parameters
+  for chosen `saveats` grid spacing and last n = `numensemble` samples in Chain. the complete set of samples
+  in the MCMC chain is returned as `fullsolution`,  refer `BPINNsolution` for more details.
+
+## Positional Arguments
+* `pde_system`: ModelingToolkit defined PDE equation or system of equations.
+* `discretization`: BayesianPINN discretization for the given pde_system, Neural Network and training strategy.
+
+## Keyword Arguments
+* `draw_samples`: number of samples to be drawn in the MCMC algorithms (warmup samples are ~2/3 of draw samples)
+* `bcstd`: Vector of standard deviations of BPINN prediction against Initial/Boundary Condition equations.
+* `l2std`: Vector of standard deviations of BPINN prediction against L2 losses/Dataset for each dependant variable of interest.
+* `phystd`: Vector of standard deviations of BPINN prediction against Chosen Underlying PDE equations.
+* `priorsNNw`: Tuple of (mean, std) for BPINN Network parameters. Weights and Biases of BPINN are Normal Distributions by default.
+* `param`: Vector of chosen PDE's parameter's Distributions in case of Inverse problems.
+* `nchains`: number of chains you want to sample
+
+# AdvancedHMC.jl is still developing convenience structs so might need changes on new releases.
+* `Kernel`: Choice of MCMC Sampling Algorithm object with appropriat args/kwargs (AdvancedHMC.jl implemenations HMC/NUTS/HMCDA).
+* `Adaptorkwargs`: A NamedTuple containing the chosen Adaptor, it's Metric and targetacceptancerate, as follows :
+    * `Adaptor`: https://turinglang.org/AdvancedHMC.jl/stable/
```

Comment:
In lines like this, can you explain it in a line and if possible link documentation specific to it instead of the repo?